src/Evolution/BoundaryCorrectionTags.hpp

```diff
@@ -0,0 +1,45 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include "DataStructures/DataBox/Tag.hpp"
+#include "NumericalAlgorithms/SpatialDiscretization/OptionTags.hpp"
+
+namespace evolution {
+namespace OptionTags {
+/// The boundary correction used for coupling together neighboring cells or
+/// applying boundary conditions.
+///
+/// In the finite volume/difference and discontinuous Galerkin literature this
+/// is ofter referred to as the "numerical flux". We avoid that nomenclature
+/// because in the discontinuous Galerkin and finite volume case it is no the
```

Comment:
I like this new paragraph, except for this typo: "no" -> "not". Same in other copy below.

Squash directly, of course.

Reply:
Done! Thank you :)